.github/workflows/release.yml

```diff
@@ -25,17 +25,19 @@ jobs:
         run: |
           npm install
           npm run build
+      - name: Extract version name
+        id: version
+        run: echo "::set-output name=version::${{ env.PLUGIN_NAME }}-$(git tag --sort version:refname | tail -n 1)"
```

Comment:
This is fine. I think the info is also available in the [github context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) so we could save the extra step?

Reply:
I'd forgotten about ref_name, but I think I still prefer working out the name/version once and reusing it in the later steps, in case we want to change the naming convention later?